pkgs/data/misc/libkkc-data/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+let
+  libkkc_version = "0.3.5";
```

Comment:
Given that you need to keep those versions synchronized with `libkkc` and it's the only user why not move this directly into `libkkc` instead?

Reply:
On another hand you could just use `libkkc` as an argument and use `libkkc.version`. Don't worry about circular dependencies -- `libkkc.version` can be used without instantiating `libkkc` derivation itself.